crates/store/src/db/models/queries.rs

```diff
@@ -481,11 +481,20 @@ pub(crate) fn upsert_accounts(
         account_id: AccountId,
     ) -> Result<Vec<Account>, DatabaseError> {
         let account_id = account_id.to_bytes();
-        let account_details_serialized =
-            SelectDsl::select(schema::accounts::table, schema::accounts::details.assume_not_null())
-                .filter(schema::accounts::account_id.eq(account_id))
-                .get_results::<Vec<u8>>(conn)?;
-        let accounts = deserialize_raw_vec::<_, Account>(account_details_serialized)?;
+        let accounts =
+            SelectDsl::select(
+                schema::accounts::table.left_join(schema::account_codes::table.on(
+                    schema::accounts::code_commitment.eq(schema::account_codes::code_commitment),
+                )),
+                (AccountRaw::as_select(), schema::account_codes::code.nullable()),
+            )
+            .filter(schema::accounts::account_id.eq(account_id))
```

Comment:
Question: should we add comments with SQL statements next to Disel code? We do this in some other places in this file, and I personally find SQL statements much easier to read.